Draft for contributing.md (#83)
pymc-devs/pymcon_web_series_website: CONTRIBUTING.md

# Contributing Guidelines

> PyMCON is a community led event and we encourage you to contribute in any form you demarcate suitable for yourself.

Reviewer: ```suggestion
PyMCon is a community led event and we encourage you to contribute in any form you demarcate suitable for yourself.
```

> Here are a few possible ways you can contribute:
> 
> - [Make a Pull request](###-Make-a-Pull-Request)
> - [Create an Issue](###-Create-an-Issue)
> - [Personal Feedback](###-Personal-Feedback)
> 

Reviewer: ```suggestion
Here are a few possible ways you can contribute to PyMCon:

- [Make a Pull request](###-Make-a-Pull-Request) to update the website.
- [Create an Issue](###-Create-an-Issue)
- [Personal Feedback](###-Personal-Feedback)
- Share the events with your network

```

Author: ah yes this is a good idea

> ### Make a Pull request

Reviewer: ```suggestion
### Make a Pull Request
```

To make a Pull Request, you can follow these steps:
- Fork the repository to your own Github Profile
- Optional: We encourage creating a specific branch in your fork to keep your workflow clean (this branch should be created from main)
- Make the necessary changes in your local codebase, check the Readme.md for more information on the website deployment.
- If you have questions about hugo or the template while making a PR then add them as comments after making the PR.
- Submit PR!

Note: We recommend raising an issue and letting it be known that you wish to make a PR (unless it's a really small change).
This is to ensure that you don't accidentally waste time on doing something that has not been previously discussed.

### Create an issue

If there is something in the website code or the repository that you think can be:
- Improved
- Corrected
- Changed

> Then create an issue.

Reviewer 2: Would be nice to link straight to issue button for convenience

### Personal Feedback

> Is there something you want to help with, and really think it should be discussed in private? Email the organizers. 

Reviewer 2: Add email for easy references

> Is there something you want to help with, and really think it should be discussed in private? Email the organizers. 

Reviewer: ```suggestion
Is there something you want to help with, and really think it should be discussed in private? [Email the organizers](https://pymcon.com/contact/). 
```

> Is there something you want to help with, and really think it should be discussed in private? Email the organizers. 

Reviewer: ```suggestion
Is there something you want to help with, and really think it should be discussed in private? Email the organizers: pymcon@pymc-devs.org 
```

Please use your own judgement to decide what is appropriate. 
A few examples of reasonable emails are - accessibility issues you want to discuss in detail personally.
While this is possible, we strongly encourage opening public issues, so everybody sees them and we can act on it sooner.

## Note

> - Please follow the [Code of Conduct](https://pymcon.com/codeofconduct/) throughout. 

Reviewer 2: Well put

- Be kind and sensitive of other people's time. 
- Try to be as informative as possible in your issues and PRs, so that you provide sufficient context. 
- For example, when creating an issue for a bug, or a PR for fixing it - 
  describe the relevant problem in detail with possible links to file or output information

## Resources

If you are not familiar with git, and/or hugo: 

- For making a PR we recommend checking the [Github Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
- For Hugo, we recommend checking [Hugo Docs](https://gohugo.io/documentation/)

> Ofcourse you can always ask questions in the Issue or PR itself as well!

Reviewer: ```suggestion
Of course you can always ask questions in the Issue or PR itself as well!
```

# Contributing Guidelines

PyMCon is a community led event and we encourage you to contribute in any form you demarcate suitable for yourself.

Here are a few possible ways you can contribute to PyMCon:

> - [Make a Pull request](###-Make-a-Pull-Request) to update the website.

Reviewer: ```suggestion
- [Make a Pull request](###-Make-a-Pull-Request) (PR) to update the website.
```
